Frontend/src/Components/Motor_Motion/Motor_Motion.js

```diff
@@ -0,0 +1,448 @@
+                        min={CONSTANTS.motor_current.MIN}
+                        max={CONSTANTS.motor_current.MAX}
+                    />
+                </GridItem>
+                <GridItem
+                    rowStart={2}
+                    colStart={4}
+                    pl='1'
+                    pr='1'
+                >
+                    <Center
+                        borderColor={borderCol}
+                        borderTop='1px'
+                        borderLeft='1px'
+                        borderRight='1px'
+                        h='100%'
+                    >
+                        <Text fontSize='sm'>Setpoints</Text>
+                    </Center>
+                </GridItem>
+                <GridItem
+                    rowStart={3}
+                    colStart={4}
+                    pl='1'
+                    pr='1'
+                >
+                    <Center
+                        bg={props.data?.crz_spd_setpt[0] ? greenBgCol : redBgCol}
+                        borderColor={borderCol}
+                        borderTop='1px'
+                        borderLeft='1px'
+                        borderRight='1px'
+                        h='100%'
+                    >
+                        <Text 
+                            as='b' 
+                            fontSize='xs'
+                            color={props.data?.crz_spd_setpt[0] ? greenTxtCol : redTxtCol}
+                        >
+                            {props.data?.crz_spd_setpt[0].toFixed(2) ?? -1.0} {CONSTANTS.crz_spd_setpt.UNIT}
+                        </Text>
+                    </Center>
+                </GridItem>
+                <GridItem
+                    rowStart={4}
+                    colStart={4}
+                    pl='1'
+                    pr='1'
+                >
+                    <Center
+                        bg={props.data?.crz_pwr_mode[0] ? greenBgCol : redBgCol}
+                        borderColor={borderCol}
+                        borderTop='1px'
+                        borderLeft='1px'
+                        borderRight='1px'
+                        borderBottom='1px'
+                        h='71.4%'
+                    >
+                        <Text 
+                            as='b' 
+                            fontSize='xs'
+                            color={props.data?.crz_pwr_mode[0] ? greenTxtCol : redTxtCol}
+                        >
+                            {props.data?.crz_pwr_setpt[0].toFixed(1) ?? -1.0} {CONSTANTS.crz_pwr_setpt.UNIT}
+                        </Text>
+                    </Center>
+                </GridItem>
+
+                <GridItem
+                    rowStart={5}
+                    colStart={4}
+                >
+                    <Center>
+                        {(props.data?.state[0] == '4') ? <Image fit={fitType} boxSize='35px' src={Images.Cruise}/> : <Box h='35px'/>}
```

Comment:
```suggestion
                        {(props.data?.state[0] == '???') ? <Image fit={fitType} boxSize='35px' src={Images.Cruise}/> : <Box h='35px'/>}
```
**The cruise state won't correspond to '4', but I'm not sure exactly what it does correspond to currently. I'm checking with firmware. When they get back to me, I'll reply to this comment with the value this should be set to.**